tee-worker/cli/src/trusted_base_cli/commands/litentry/request_vc.rs

```diff
@@ -505,17 +508,33 @@ impl RequestVcCommand {
 		.sign(&KeyPair::Sr25519(Box::new(alice)), nonce, &mrenclave, &shard)
 		.into_trusted_operation(trusted_cli.direct);
 
-		match perform_trusted_operation::<RequestVCResult>(cli, trusted_cli, &top) {
-			Ok(mut vc) => {
-				let decrypted = aes_decrypt(&key, &mut vc.vc_payload).unwrap();
-				let credential_str = String::from_utf8(decrypted).expect("Found invalid UTF-8");
-				println!("----Generated VC-----");
-				println!("{}", credential_str);
-			},
-			Err(e) => {
-				println!("{:?}", e);
-			},
+		if !self.isolated {
+			match perform_trusted_operation::<RequestVCResult>(cli, trusted_cli, &top) {
+				Ok(mut vc) => {
+					let decrypted = aes_decrypt(&key, &mut vc.vc_payload).unwrap();
+					let credential_str = String::from_utf8(decrypted).expect("Found invalid UTF-8");
+					println!("----Generated VC-----");
+					println!("{}", credential_str);
+				},
+				Err(e) => {
+					println!("{:?}", e);
+				},
```

Comment:
It seems we can avoid the repetition by something like:
```
let maybe_vc = if isolated { .. } else { .. }
match maybe_vc ...
```

Reply:
Updated ✅